src/actions/email-actions.js

```diff
@@ -132,8 +132,10 @@ export const saveEmailTemplate = (entity, noAlert = false) => async (dispatch, g
             entity
         )(params)(dispatch)
             .then((payload) => {
-                if (!noAlert)
+                if (!noAlert){
                     dispatch(showSuccessMessage(T.translate("emails.template_saved")));
+                    dispatch(getEmailTemplate(entity.id));
```

Comment:
is not the payload already updated with the latest info ?
its really necessary to perform another request ?, i do think that the PUT returns the entity updated

Reply:
the PUT is missing the `versions` prop

Reply:
should be fixed
if u add this action 
https://github.com/fntechgit/summit-admin/blob/36064e9d7969ec5177d2bf690ff8521e1e6a884b/src/actions/email-actions.js#L40
on the reducer here[
](https://github.com/fntechgit/summit-admin/blob/36064e9d7969ec5177d2bf690ff8521e1e6a884b/src/reducers/emails/email-template-reducer.js#L82)
the the extra api call will not be necessary

Reply:
also recall to put the proper expand here
https://github.com/fntechgit/summit-admin/blob/36064e9d7969ec5177d2bf690ff8521e1e6a884b/src/actions/email-actions.js#L122
expand: 'parent,versions'